test/SPI/resilience.swift

```diff
@@ -1,29 +0,0 @@
-// Test limits of SPI members in frozen types.
```

Comment:
This file was deleted because the tests it contains seem to be completely superseded by the `spi_members.swift` test.